starfish/spots/_detector/pixel_spot_detector.py

```diff
@@ -96,6 +106,16 @@ def add_arguments(cls, group_parser):
         group_parser.add_argument(
             "--area-threshold", type=float, default=2, help="minimum area of a feature"
```

Comment:
We also churned on this -- our decision was that we don't have enough scientific consensus to give good defaults, so we wouldn't provide any in those cases. I think the API does that properly here, and the CLI should match that.